sound/lame/Makefile

```diff
@@ -63,8 +63,8 @@ TARGET_CFLAGS+=-msse
 endif
 
 ifeq ($(CONFIG_LAME-LIB_OPTIMIZE_SPEED),y)
+	TARGET_CFLAGS := $(filter-out -O%,$(TARGET_CFLAGS))
 	TARGET_CFLAGS += $(TARGET_CFLAGS) -O3 -ffast-math
```

Comment:
Should be:
```
TARGET_CFLAGS += -O3 -ffast-math
```
additional ref to `$(TARGET_CFLAGS)` is redundant.

Reply:
Well, yes as it overrides the previous variables but it's ugly.
Here's a neater version and used in other packages.